compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -212,7 +236,7 @@ FailureOr<Flow::DispatchRegionOp> Flow::appendDispatchRegionResult(
 }
 
 Flow::DispatchRegionOp Flow::makeDispatchRegionWithWorkload(
-    OpBuilder &builder, Location loc, Optional<ValueRange> workload) {
+    OpBuilder &builder, Location loc, Optional<SmallVector<Value>> workload) {
```

Comment:
do we really need `workload` be Optional type? `SmallVector<Value>` is good enough to me.

Reply:
Good point. Fixed.